app/build.gradle

```diff
@@ -33,10 +34,14 @@ android {
 }
 
 dependencies {
+
     implementation fileTree(dir: "libs", include: ["*.jar"])
 
     implementation(project(":baseandroid"))
 
     implementation 'com.android.volley:volley:1.1.0'
-    implementation 'com.github.aabhasr1:OtpView:v1.1.2-ktx' // kotlin
+    implementation 'com.github.aabhasr1:OtpView:v1.1.2-ktx'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
```

Comment:
It's already exposed from `baseandroid`.

If not add it there with transitive dependency.